perl-lib/OESS/lib/OESS/DB/Node.pm

```diff
@@ -32,19 +32,7 @@ sub fetch{
     
     return if(!defined($node) || !defined($node->[0]));
 
-    $node = $node->[0];
-
-    my $res = $db->execute_query("select interface.interface_id from interface natural join interface_instantiation where interface.node_id = ? and interface_instantiation.end_epoch = -1",[$node->{'node_id'}]);
-    
-    my @ints;
-
-    foreach my $int_id (@$res){
-        my $int = OESS::Interface->new( db => $db, interface_id => $int_id);                                                                                                                                        
-        push(@ints, $int);
-    }
-
-    $node->{'interfaces'} = \@ints;
-    return $node;
+    return $node->[0];
```

Comment:
what am I missing here... seems like this would be a problem in other chunks of the code am I wrong?

Reply:
If you look back in history you see that a node's interfaces were never being properly loaded as the required node_id was never passed to the query. This is what allowed us to avoid a lookup look in the first place.
https://github.com/GlobalNOC/OESS/blob/a1122825aaae29d7133a52fe48fd8c19ad9ed21e/perl-lib/OESS/lib/OESS/DB/Node.pm#L31

Reply:
Jonathan and I discussed, this should be fine for now, as it isn't used